packages/core/admin/admin/src/content-manager/components/DynamicTable/ConfirmDialogPublishAll/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Dialog, DialogBody, DialogFooter, Flex, Typography, Button } from '@strapi/design-system';
+import { ExclamationMarkCircle, Check } from '@strapi/icons';
+import InjectionZoneList from '../../InjectionZoneList';
+import { getTrad } from '../../../utils';
+
+const ConfirmDialogPublishAll = ({ isConfirmButtonLoading, isOpen, onToggleDialog, onConfirm }) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Dialog
+      onClose={onToggleDialog}
+      title={formatMessage({
+        id: 'app.components.ConfirmDialog.title',
+        defaultMessage: 'Confirmation',
+      })}
+      labelledBy="confirmation"
+      describedBy="confirm-description"
+      isOpen={isOpen}
+    >
+      <DialogBody icon={<ExclamationMarkCircle />}>
+        <Flex direction="column" alignItems="stretch" gap={2}>
+          <Flex justifyContent="center">
+            <Typography id="confirm-description">
+              {formatMessage({
+                id: getTrad('popUpWarning.bodyMessage.contentType.publish.all'),
+                defaultMessage: 'Are you sure you want to publish these entries?',
+              })}
+            </Typography>
+          </Flex>
+          <Flex>
+            <InjectionZoneList area="contentManager.listView.publishModalAdditionalInfos" />
+          </Flex>
```

Comment:
Im not sure about wrapping the InjectionZone in a Flex – wouldn't that mean if I passed an array of children for the injection it would line them up `1 | 2 | 3` by default?

Reply:
I'm not sure it would because InjectionZoneList wraps injected components in a ul tag. But the flex isn't useful I'll remove it 👍